ene_008_oil_chemical_spill_incidents/contents/src/__init__.py

```diff
@@ -155,117 +225,140 @@ def processData(existing_ids):
                         new_rows[-1][-1] = new_rows[-1][-1] + ' ' + row[0].replace('\t', '')
                     # If several things are in the row, the break was probably mid-row.
                     elif len(row) > 1 and len(row) < 17:
-                        # finish the last desciption
+                        # finish the last description of the last row in the list
                         new_rows[-1][-1] = new_rows[-1][-1] + ' ' + row[0].replace('\t', '')
                         # append other items to row
                         new_row = new_rows[-1]
+                        # need to process one less column for last row since description is already filled
                         offset_factor = len(new_rows[-1])-1
+                        # go through each column in the Carto table
                         for field in CARTO_SCHEMA:
+                            # skip if the field is 'uid' or 'the_geom'
                             if field == 'uid' or field == 'the_geom':
                                 continue
                             try:
+                                # find the appropriate location for the field by subtracting offset
                                 loc=idx[field]-offset_factor
                                 if loc>0:
+                                    # insert value for columns to the new_row list 
+                                    # if the column is empty, insert None
                                     val = row[loc] if row[loc] != '' else None
                                     new_row.append(val)
                             except IndexError:
                                 pass
+                        # check if last row in new_rows matches with the new_row list
                         new_rows[-1]==new_row
```

Comment:
I don't think this is doing anything right?
```suggestion
```

Reply:
right. the author was probably debugging it. so, we can get rid of it.